### PR TITLE
Make minimum staking amount as parameter

### PR DESCRIPTION
**Pull request by Jianzhuo**
close issue Fix the default minumStakingAmount #84

### CODE REVIEW

**Comment by mzxyz** [comment]
Only merge this PR after #86 merged and upgraded

**Review by mzxyz — APPROVED** [review]

File: scripts/config/local.config.ts
Comment (mzxyz) [inline scripts/config/local.config.ts]:
```suggestion
            IndexerRegistry: [1000e18],
```

File: test/setup.ts
Comment (mzxyz) [inline test/setup.ts]:
```suggestion
            IndexerRegistry: [1000e18],
```

**Review by mzxyz — APPROVED** [review]

**Comment by mzxyz** [comment]
## TODO: 

- [ ] Fix the test cases

File: scripts/config/testnet.config.ts
Comment (mzxyz) [inline scripts/config/testnet.config.ts:17]: Why not use `1000e18`, I think it would be more readable

**Review by mzxyz — APPROVED** [review]
Just one question, LGTM

File: scripts/config/testnet.config.ts
Comment (Jianzhuo) [reply scripts/config/testnet.config.ts:17]: 1e18 will cause overflow to convert to BigNumber

**Pull request merged**
